classroom_in_a_box/ks/\

```diff
@@ -0,0 +1,80 @@
+#platform=x86, AMD64, or Intel EM64T
```

Comment:
Looks to me like what happens why I mistype a vim command and accidentally save a copy of the file with a weird name.